include/casmutils/xtal/coordinate.hpp

```diff
@@ -10,59 +10,27 @@ namespace xtal
 {
 
 class Lattice;
-
-// TODO: Just erase this class, it's super annoying. You can do all of this
-// stuff with functions
-
+namespace coordinate
```

Comment:
Do we need a nested namespace for this?

Reply:
I remember you suggesting putting all these functions in a namespace. Hence, I used it I suppose. I will just get rid of it, no issues.

Reply:
I'm not opposed to the namespace, but I would shorten it. Unless you think it's better without it altogether.